Docsite: add Community guide (#93)
ansible/ansible-sdk: docs/source/index.rst

Ansible SDK is a toolkit that lets you harness the power and simplicity of Ansible automation directly from your applications.

.. note::

>   Need help or want to discuss all things related to Ansible SDK? See the :ref:`Community guide<community>` and join the conversation!

Reviewer: ```suggestion
   Need help or want to discuss all things related to Ansible SDK? See the :ref:`Community guide<community>` and join the conversation!
```

>   Need help or want to discuss Ansible SDK including the documentation? See the :ref:`Community guide<community>` to learn how to join the conversation!

Reviewer: ```suggestion
  Need help or want to discuss all things related to Ansible SDK? See the :ref:`Community guide<community>` and join the conversation!
```
